features/step_definitions/direct_award_steps.rb

```diff
@@ -1,6 +1,6 @@
 When(/^I have created and saved a search$/) do
```

Comment:
It might be nice to parameterise this with the name to save as, which means the Scenarios can set up searches with different names, e.g. `I have created and saved a search called 'my other cloud project'`

It could even use the name passed in as the search query too, e.g. `I have created and saved a search for 'email analysis provider'`' - and then different scenarios can use different queries and have searches saved with different names.  This multi-step method would then be less of a black box in terms of what it creates for the Scenarios (at the moment the Scenarios assume 'my cloud project' as a name, but it isn't obvious from reading the scenarios on their own where that name comes from - by parameterising it I think it will make the scenario files easier to follow.